CraftTweaker2-API/src/main/java/crafttweaker/api/oredict/IOreDict.java

```diff
@@ -9,13 +9,20 @@
  * @author Stan
  */
 @ZenClass("crafttweaker.oredict.IOreDict")
+@IterableSimple("crafttweaker.oredict.IOreDictEntry")
 @ZenRegister
-public interface IOreDict {
+public interface IOreDict extends Iterable<IOreDictEntry> {
+
     
-	@ZenMethod
-	@ZenMemberGetter
+    @ZenMemberGetter
+    @ZenOperator(OperatorType.INDEXGET)
+    @ZenMethod
     IOreDictEntry get(String name);
-    
+
     @ZenGetter("entries")
     List<IOreDictEntry> getEntries();
+
+    @ZenOperator(OperatorType.CONTAINS)
+    @ZenMethod
+    boolean contains(String name);
```

Comment:
Check if an oreDict entry already exists (without the fear of creating a new one)